src/platform/silabs/wifi/SiWx/WifiInterfaceImpl.cpp

```diff
@@ -321,7 +315,6 @@ sl_status_t sl_wifi_siwx917_init(void)
 }
 
 // TODO: this changes will be reverted back after the Silabs WiFi SDK team fix the scan API
```

Comment:
Can this todo be removed?

Reply:
Removed To-Do comment as scan call back is giving the mapped security type. Earlier it was mapping in our code based on wifi sdk sec type.